### ISSUE TITLE
Warning

### ISSUE THREAD

**Issue by DevilJamJar** [xanthe1337/iLinked-Selfbot-#1]
**TL;DR THIS IS A SCAM, DON'T USE IT OR GET YOUR ACCOUNT HACKED.**

Real selfbots would never require windows defender to be off.... The selfbot may or may not work correctly by itself but by turning off your windows defender you're practically sending this guy your token.

Just don't do it... It's an obvious scam and it doesn't take a genius to decode the .exe file to figure out that once you run this with your token in your account is basically hijacked and this guy has full control of it, even if you change your password.

This guy xanthe already runs multiple spam discord bots pretending to be Discord and that you've 'won a free nitro' etc. All of these are either scams to join their servers or to give them your account token. Do not trust anything by this guy.

**Comment by coats1337** [comment]
my guy, they're impersonating me lol

**Comment by coats1337** [comment]
but ur also right
